packages/arb-token-bridge-ui/src/hooks/useArbTokenBridge.ts

```diff
@@ -41,7 +41,7 @@ export const wait = (ms = 0) => {
   return new Promise(res => setTimeout(res, ms))
 }
 
-function isClassicL2ToL1TransactionEvent(
+function isClassicChildToParentTransactionEvent(
```

Comment:
i am a bit conflicted with this because classic was only used by Arb One so might as well just keep it as it was